src/oph/ehoks/oppija/schema.clj

```diff
@@ -14,14 +15,18 @@
 (s/defschema
   Jakolinkki
   "Tutkinnon osan jakolinkki"
-  {:share-id java.util.UUID
-   :tutkinnonosa-module-uuid java.util.UUID
-   :tutkinnonosa-tyyppi s/Str
-   :shared-module-uuid java.util.UUID
-   :shared-module-tyyppi s/Str
-   :voimassaolo-alku LocalDate
-   :voimassaolo-loppu LocalDate
-   :hoks-eid s/Str})
+  (s/->Either
+    [{:oppija-nimi s/Str
+      :oppija-oid s/Str
+      :tutkinto-nimi Translated
+      :osaamisala-nimi Translated
+      :voimassaolo-alku LocalDate,
+      :voimassaolo-loppu LocalDate
+      :module-tyyppi s/Str
+      :module s/Any
+      :tutkinnonosa-tyyppi s/Str
+      :tutkinnonosa s/Any}
```

Comment:
Lisäisin tähän modulen tilalle osaamisen-osoittaminen ja osaamisen-hankkimistapa kentät joiden tyyppi olisi hoks.schemasta otettu jolloin frontissa olisi helppo sijoittaa olemassaoleviin modeleihin. Mutta tosiaan tämä skeema tulee varmasti vielä elämään kunhan fronttia lähdetään tekemään ja selviää tarkemmin tarpeet.

Reply:
Jep, nuo Any:t ovat kyllä ihan lipsahdus, tarkoitus oli käyttää tässä juurikin niitä olemassaolevia skeemoja jotta frontissa ei tarvitse keksiä pyörää uudestaan. Pitää vaan tehdä tähän viritys jotta module ja tutkinnonosa voivat pitää sisällään erityyppisiä sisältöjä.

Reply:
Tekisin niin että poistaisin modulen ja lisäisi kaksi uutta kenttä osaamisen-osoittaminen ja osaamisen-hankkimistapa. Frontti voi vaan sitten välittää molemmat komponentille ja sen rendaa sen missä dataa sattuu olemaan.

Reply:
Tämä on nyt muutettu näin, toisessa on dataa ja toinen on nil.